superset/config.py

```diff
@@ -1498,13 +1490,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 
 # Global async query config options.
 # Requires GLOBAL_ASYNC_QUERIES feature flag to be enabled.
-GLOBAL_ASYNC_QUERIES_REDIS_CONFIG = {
```

Comment:
These were the same as the [defaults](https://github.com/redis/redis-py/blob/master/redis/client.py#L144-L187) and thus unncessary.